packages/www/components/Dashboard/AssetDetails/EmbedVideoDialog.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+  Dialog,
+  DialogContent,
+  DialogTitle,
+  Heading,
+  Flex,
+  Button,
+  Text,
+  TextField,
+  useSnackbar,
+} from "@livepeer/design-system";
+import { Asset } from "livepeer";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+
+const embedStringForAsset = (playbackId: string) => `
+<iframe
+src="https://lvpr.tv?v=${playbackId}"
```

Comment:
Regarding linking to the lvpr.tv player, I think the only real problem from it not using the livepeer.js player yet is that it will not publish any viewership metrics when used, which IMO is highly desired feature. We should update it ASAP to use the SDK player instead. 

@adamsoffer I mentioned today that I could update the player to use the SDK player, but after the day today I'm actually not sure when I'll have bandwidth to do that. It'd also involve setting up React from scratch on the project (hopefully a really minimal version of it) and ideally find a way to download/embedded it statically on the IPFS directory, not to have the IPFS website depending on external resources. Do you think anyone from UX team could pick up on that? I can provide all support on the existing stuff in the project, deploying on fleek, etc

Reply:
No problem! I can pick that up 👍